jdaviz/configs/cubeviz/plugins/parsers.py

```diff
@@ -38,6 +39,13 @@ def parse_data(app, file_obj, data_type=None, data_label=None, parent=None):
         The data type used to explicitly differentiate parsed data.
     data_label : str, optional
         The label to be applied to the Glue data component.
+    parent : str, optional
+        Data label for "parent" data to associate with the loaded data as "child".
+    cache : None, bool, or str
+        Cache the downloaded file if the data are retrieved by a query
+        to a URL or URI.
+    local_path : str, optional
```

Comment:
I wonder if we want to use a more generic `**kwargs` to future-proof the API but I guess this is okay for now.

Reply:
I know you know this but while it's definitely future-proof, it's also non-specific and can cause user heartache. I don't take a specific stance, other than agreeing that 
> this is okay for now

Reply:
I should have clarified: I specifically meant in a future where we start to support Cloud FITS etc, and need to introduce a bunch of new keywords that only apply under certain conditions, like `local_path`.

That said... I think we already need to add `timeout`.